### PR TITLE
docs: Improve Run Novu locally docs

### PR DESCRIPTION
**Pull request by npentrel**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Docs updates for the Run Novu locally page.

- **Why was this change needed?** (You can also link to an open issue here)
Improve Readability.

- **Other information**:

### CODE REVIEW

File: docs/docs/community/run-locally.md [hunk 1]
```diff
@@ -86,36 +86,37 @@ If you want to test certain parts of Novu or run it in production mode though, s
...
 
-Run the cypress test suite using:
+Run the cypress test suite with the following command:
 
 ```shell
 cd apps/web && npm run cypress:run
 ```
 
-To open the cypress management windows to debug tests run:
+To open the cypress management windows to debug test runs, run with the following commands:
```
Comment (p-fernandez) [inline docs/docs/community/run-locally.md:119]:
```suggestion
To open the cypress management windows to debug tests, run with the following commands:
```

**Review by p-fernandez — APPROVED** [review]
🌟

**Comment by p-fernandez** [comment]
Please fix the merge conflict so we can run the integration checks. Thank you!

**Comment by npentrel** [comment]
All good to go I think!

**Pull request merged**